mlkem/indcpa.c

```diff
@@ -357,49 +389,48 @@ void mlk_indcpa_keypair_derand(uint8_t pk[MLKEM_INDCPA_PUBLICKEYBYTES],
    */
   MLK_CT_TESTING_DECLASSIFY(publicseed, MLKEM_SYMBYTES);
 
-  mlk_gen_matrix(a, publicseed, 0 /* no transpose */);
+  mlk_gen_matrix(pk->at, publicseed, 0 /* no transpose */);
```

Comment:
Can you leave a comment that the matrix will be transposed later? Otherwise, it's confusing why the variable is named `at`.